intro-Linux/ch_uploading_files.tex

```diff
@@ -2,25 +2,29 @@ \chapter{Uploading/downloading/editing files}
 
 \subsection{Uploading/downloading files}
 
-Detailed information about uploading/downloading files with ``scp'' (using an OS
-X or Linux client) or WinSCP (Windows client), see
-\url{http://hpc.ugent.be/userwiki/index.php/User:VscCopy}.
+To transfer files from and to the HPC, see
+\href{\HPCManualURL#sec:filetranfer}{the section about tranferring files in the HPC manual}.
+
+\ifwindows
 
 After copying files it is advised to run
 
 \begin{prompt}
 %\shellcmd{dos2unix filename}
 \end{prompt}
 
-as this will fix any problems with windows / unix conversion.
+as this will fix any problems with Windows/Unix conversion.
 
+\fi
 \section{Symlinks for data/scratch}
 
 As we end up in the home directory when connecting, it would be convenient if we
-could access our data and vo storage. To facilitate this we shall create
-symlinks to them in our home directory.
+could access our data and VO storage. To facilitate this we will create
+symlinks to them in our home directory. This will create 4 symbolic links
+(they're like ``shortcuts'' on your desktop \ifwindows
+and they look like directories in WinSCP\fi)
```

Comment:
I assume you made sure this renders correctly in both the Windows & non-Windows versions?

Reply:
I tested it, apart from having an extra space before the closing paren, this looks okay.

Reply:
This is nitpicking, but can we avoid the extra space? It'll look sloppy... Is using `desktop\ifwindows<space>and` an issue?

Reply:
Yes, that is an option, but renders to `desktopand` on Windows... I fixed it by putting it in an if-else now.

Reply:
How about forcing a space? `desktop\ifwindows~and`?